charts/enterprise-kyverno-operator/templates/_cr-kyverno-helpers.tpl

```diff
@@ -0,0 +1,74 @@
+{{- define "enterprise-kyverno.kyveroDefaultHelm" -}}
```

Comment:
moved code from cr-kyverno to a template function in order to be able to merge the fixed params and override same sections via commandline.